bin/gftools-add-axis.py

```diff
@@ -102,6 +102,10 @@ def main(font: str):
     axis_proto.max_value = fvar_axis.maxValue
     # Input values for this axis must aligned to 10^precision
     axis_proto.precision = 1  # ask user?
+    # Some parameter I don't know what it does
+    axis_proto.fallback_only = false
+    # Illustration path
+    axis_proto.illustration_url = '' # leave empty? 
```

Comment:
Wouldn't it be better if the tool could take an illustration as an arg?

Reply:
We probably do the illustration in a second step once this axis is already in sandbox

Reply:
Arg could be optional.